packages/test/src/test-retry-policy.ts

```diff
@@ -35,10 +35,17 @@ test('compileRetryPolicy validates backoffCoefficient is greater than 0', (t) =>
   });
 });
 
-test('compileRetryPolicy validates maximumAttempts is greater than 0', (t) => {
```

Comment:
Why did you delete this test?

Reply:
100% accidental. Sorry about that and thanks for catching this. Fixed :+1: